local_test/run_worker.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+set -ex
+
+mkdir -m700 -p /root/.ssh
+cp local_test/id_rsa.rrrspec /root/.ssh/id_rsa
```

Comment:
Added `chmod 600` and `-p` flag.
https://github.com/cookpad/rrrspec/commit/8c76d06de45e6fb7093f84d6898b9d9fe5ab06c0